recipes/protobuf/all/conanfile.py

```diff
@@ -38,13 +39,24 @@ def source(self):
         extracted_folder = self.name + "-" + self.version
         os.rename(extracted_folder, self._source_subfolder)
 
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
     def configure(self):
-        if self.settings.os == "Windows" and self.settings.compiler == "Visual Studio":
+        if self.options.shared:
             del self.options.fPIC
+
             compiler_version = Version(self.settings.compiler.version.value)
             if compiler_version < "14":
                 raise ConanInvalidConfiguration("On Windows Protobuf can only be built with "
                                                 "Visual Studio 2015 or higher.")
+            if self.options.shared and "MT" in self.settings.compiler.runtime:
```

Comment:
I think runtime needs to be stringified.
```suggestion
            if self.options.shared and "MT" in str(self.settings.compiler.runtime):
```

Reply:
I just tested it locally with a dummy `if "MT" in self.settings.compiler.runtime` condition and it seems to work correctly

Reply:
It detects `MT/MTd` correctly?

Reply:
Yes